language_detector/main.py

```diff
@@ -1,8 +1,21 @@
 # -*- coding: utf-8 -*-
 
-"""This is the entry point of the program."""
-
 def detect_language(text, languages):
-    """Returns the detected language of given text."""
-    # implement your solution here
-    pass
+    
+    globalcount = 0
+    lang = ""
+    
+    for dictionary in languages:
+        
+        count = 0
+        for word in text.split():
+            
+            if word in dictionary['common_words']:
+                count += 1
+                    
+        if count > globalcount:
+            globalcount = count
+            lang = dictionary['name']
```

Comment:
Good solution overall. The general idea was same with ours in group 4, having two for loops. one for iterating through the languages in the language dictionary and the other for the words in users text. then creating a counter for the languages as the occur. In the end returning the one with the highest. 
Just little differences in our implementations and choice of specific python built-in functions.
for example, we used `max`: to check for the language that occured most, here:
``` python
 return(max(dictionar_of_languages, key=dictionar_of_languages.get))
```
while you guys simply took care of the max issue during the for loop, by assigning it to `lang` variable, whenever the count was greater than global count:
```python
 if count > globalcount:
            globalcount = count
            lang = dictionary['name']
 ```
and then returning `lang`, 

```python 
return lang
```
which is equivalent to our `max` return.